src/utilities/fixupx.go.tmpl

```diff
@@ -0,0 +1,27 @@
+{{/*
+	Replies with fixupx.com links to messages containing x.com links.
+
+  See <https://yagpdb-cc.github.io/utilities/fixupx> for more information.
```

Comment:
If it's not too much trouble, would you mind reindenting this file to consistently use tabs instead of spaces? It's a little inconsistent as is.